### PR TITLE
ci: added bug report issue form

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [x] Other... Please describe: ci

## What is the new behavior?

image

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Review by nsbarsukov — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              - 'Yes'
+              - 'No'
+      validations:
+          required: true
+
+    - type: input
+      id: reproduction-url
+      attributes:
+          label: Please provide a link to a minimal reproduction of the bug
+          value: https://stackblitz.com/edit/taiga-starter?file=src%2Fapp%2Fapp.component.html
```
Comment (waterplea) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:44]: Let's remove the part after '?'

Comment (splincode) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml]: fixed)

**Review by waterplea — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov check it, please)

**Comment by splincode** [comment]
@nsbarsukov check it, please)

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**